frontend/src/static/js/api.js

```diff
@@ -6,17 +6,28 @@ const REPORT_DIR = '.';
 // data retrieval functions //
 function loadJSON(fname) {
   if (window.REPORT_ZIP) {
-    return window.REPORT_ZIP.file(fname.slice(2)).async('text')
-      .then(txt => JSON.parse(txt));
+    const zipObject = window.REPORT_ZIP.file(fname.slice(2));
+    if (zipObject) {
+      return zipObject
+          .async('text')
+          .then((txt) => {
+            try {
+              return JSON.parse(txt);
+            } catch (err) {
+              return Promise.reject(new Error('JSON file is invalid.'));
```

Comment:
Still don't see the pop-up though :thinking:
Seems like the code doesn't reach this `catch` section at all, perhaps we need a different way of catching it.

Anyway, since this is a `p.High` and we are taking too much time on this (and other delays), let's just remove this portion first and raise an issue afterwards to fix this error (can leave the other error handling portion).

Reply:
Remove this portion first, since it doesn't seem do anything right now (doesn't reach this portion of code apparently)